great_expectations/data_context/data_context/file_data_context.py

```diff
@@ -128,7 +128,7 @@ def _init_variables(self) -> FileDataContextVariables:
         )
         return variables
 
-    def _save_project_config(self) -> None:
+    def _save_project_config(self, fds_datasource=None) -> None:
```

Comment:
should `SerializableDataContext` be updated accordingly?

Reply:
Yes